src/session.rs

```diff
@@ -140,6 +140,9 @@ impl Session {
         if let Some(cursormode) = options.cursor_mode {
             self.cursor_mode = cursormode;
         }
+        if let Some(cursor_mode) = options.cursor_mode {
```

Comment:
haaa.. these days I finally realize that this problem is already solved some days. but still thank you very much. and you can see the line you added is already existed above. so you should remove this part